### PR TITLE
Quote element

### PR DESCRIPTION
**Pull request by amitaibu**
| Mobile | Tablet | Desktop | RTL | Node view | Node edit |
| -- | -- | -- | -- | -- | -- |
| image | image | image | image | image | image |

### CODE REVIEW

File: web/modules/custom/server_migrate/config/install/migrate_plus.migration.paragraph_quote.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    - plugin: migration_lookup
+      migration:
+        - media_image
+      no_stub: true
+
+migration_dependencies:
+  required:
+    - media_image
+
+
```
Comment (amitaibu) [inline web/modules/custom/server_migrate/config/install/migrate_plus.migration.paragraph_quote.yml:38]:
```suggestion
```

**Comment by amitaibu** [comment]
@mgurjanov fyi ^^

cc @nora-novak @isapisa

**Review by andreytroeglazov — APPROVED** [review]

**Comment by amitaibu** [comment]
- Added [maxlength](https://www.drupal.org/project/maxlength) module
- Quote text is limited to 280 chars (soft limit)
- Quote image height on `lg` and `xl` is limited to 400px, so if text is short, the image doesn't over-grow

| Short text | 280 chars |
| -- | -- |
| image | image |

File: web/modules/custom/server_general/src/Plugin/EntityViewBuilder/ParagraphQuote.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+   *   Render array.
+   */
+  public function buildFull(array $build, ParagraphInterface $entity): array {
+
+    /** @var \Drupal\media\MediaInterface $media */
+    $media = $this->getReferencedEntityFromField($entity, 'field_image');
+    $image_credit = $this->getTextFieldValue($media, 'field_media_credit');
+
+    $element = $this->buildElementQuote(
+      $this->buildMediaResponsiveImage($entity, 'field_image', 'hero'),
```
[the remark this answers is not in the document]
Comment (amitaibu) [reply web/modules/custom/server_general/src/Plugin/EntityViewBuilder/ParagraphQuote.php]: Done

**Pull request merged**